tests/http/src/test/resources/http-listener-error-interpretation-config.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mule xmlns="http://www.mulesoft.org/schema/mule/core"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:httpn="http://www.mulesoft.org/schema/mule/httpn"
+      xsi:schemaLocation="
+               http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+               http://www.mulesoft.org/schema/mule/httpn http://www.mulesoft.org/schema/mule/httpn/current/mule-httpn.xsd">
+
+    <httpn:listener-config name="listenerConfig">
+        <httpn:listener-connection host="localhost" port="${port}"/>
+    </httpn:listener-config>
+
+    <httpn:request-config name="requestConfig">
+        <httpn:request-connection host="localhost" port="${port2}"/>
+    </httpn:request-config>
+
+    <flow name="customErrorType">
+        <httpn:listener config-ref="listenerConfig" path="/error"/>
+        <custom-processor class="org.mule.test.module.http.functional.listener.HttpListenerErrorInterpretationTestCase$ErrorProcessor"/>
```

Comment:
Can all custom-processor be migrated to test:component?. Since we are going to get rid of custom-processor for GA.